docs/man-pages/condor_store_cred.rst

```diff
@@ -110,6 +110,8 @@ Options
     action.  If used with the *query-oauth* action, makes sure that
     the same audience was requested in the original credential.
     Requires credentials to be in JSON format.
+ **-P**
+    Indicate that the Oauth2 credential is an access token.
```

Comment:
Seems like we should mention that the default is a refresh token a little further up in the man page.

Should we add the converse flag indicating that the OAuth2 credential is a refresh token?

Reply:
I added some text about -P in the paragraph above for oauth credentials.